website/docs/cloud-docs/policy-enforcement/define-policies/prewritten-sentinel.mdx

```diff
@@ -0,0 +1,129 @@
+
+- [Sentinel documentation](/sentinel/docs) 
+- README documentation included with each of the policy libraries.
+
+Complete the following steps to implement pre-written Sentinel policies in your workspaces:
+
+1. Obtain the policies you want to implement: You download policies directly into your repository or create a fork of the HashiCorp repositories. Alternatively, you can add the Terraform module to your configuration, which acquires the policies and connects them to your workspaces in a single step.
+1. Connect policies to your workspace. If you downloaded policies or forked one or more repositories, you must manually connect the repository containing the policies to your HCP Terraform or Terraform Enterprise workspaces.       
+
+## Requirements
+
+You must use one of the following Terraform applications:
+
+- HCP Terraform
+- Terraform Enterprise v202312-1 or newer
+
+### Permissions
+
+Your HCP Terraform or Terraform Enterprise user account must either be a member of the owners team or have the **Manage Policies** organization-level permissions enabled to create new policy sets and policies. Refer to the following topics for additional information:
+
+- [Organization owners](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners)
+- [Manage policies](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policies)
+
+### Version control system
+
+You must have a GitHub account connected to HCP Terraform or Terraform Enterprise to manually connect policy sets to your workspaces. Refer to [Connecting VCS Providers](/terraform/cloud-docs/vcs) for instructions.
+
+## Obtain policies
+
+You can use the policy libraries created and maintained by HashiCorp. The libraries are stored in the following GitHub repositories:
+
+- [policy-library-cis-aws-efs-terraform](https://github.com/hashicorp/policy-library-cis-aws-efs-terraform)
+- [policy-library-cis-aws-rds-terraform](https://github.com/hashicorp/policy-library-cis-aws-rds-terraform)
+- [policy-library-cis-aws-vpc-terraform](https://github.com/hashicorp/policy-library-cis-aws-vpc-terraform)
+- [policy-library-cis-aws-iam-terraform](https://github.com/hashicorp/policy-library-cis-aws-iam-terraform)
+- [policy-library-cis-aws-s3-terraform](https://github.com/hashicorp/policy-library-cis-aws-s3-terraform)
+- [policv-library-cis-aws-cloudtrail-terraform](https://github.com/hashicorp/policv-library-cis-aws-cloudtrail-terraform)
+- [policy-library-cis-aws-kms-terraform](https://github.com/hashicorp/policy-library-cis-aws-kms-terraform)
+- [policy-library-cis-aws-ec2-terraform](https://github.com/hashicorp/policy-library-cis-aws-ec2-terraform)
+
+Use one of the following methods to obtain pre-written policies:
+
+- Download policies from the registry
+- Fork the HashiCorp policy GitHub repository
+- Add the Terraform module to your configuration 
```

Comment:
```suggestion
- **Download policies from the registry**:  Use this method if you want to assemble custom policy sets without customizing policies.  
- **Fork the HashiCorp policy GitHub repository**: Use this method if you intend to customize the policies. 
- **Add the Terraform module to your configuration**: Use this method to implement specific versions of the policies as-is. This method also connects the policies to workspaces in the Terraform configuration file instead of connecting them as a separate step. 
```